datalad/customremotes/ria_utils.py

```diff
@@ -204,3 +206,8 @@ def create_ds_in_store(io, base_path, dsid, obj_version, store_version):
     io.mkdir(archive_dir)
     io.mkdir(dsobj_dir)
     io.write_file(version_file, obj_version)
+    if alias:
+        alias_dir = base_path / "alias"
+        if not io.exists(alias_dir):
```

Comment:
Conditional shouldn't be necessary. `io.mkdir` methods don't fail on existing dirs.

Reply:
Cool, I dropped the conditional.